app/Resources/assets/js/lsdoc/index.js

```diff
@@ -649,3 +649,10 @@ $(window).resize(function(){
     $('#treeSideLeft').height($("#treeView").height());
 });
 
+
+$(".export_csv_comment").click(function(){
```

Comment:
This block should be a function in the comments.js file

Reply:
Moved code to comment.js file